.github/workflows/tests.yaml

```diff
@@ -3,6 +3,11 @@ name: tests
 
 env:
   GINKGO_VERSION: v1.15.2
+  # DEBUG WAIT
+  TMATE_SERVER_HOST: ${{ secrets.TMATE_SERVER_HOST }}
+  TMATE_SERVER_PORT: ${{ secrets.TMATE_SERVER_PORT }}
+  TMATE_SERVER_RSA_FINGERPRINT: ${{ secrets.TMATE_SERVER_RSA_FINGERPRINT }}
+  TMATE_SERVER_ED25519_FINGERPRINT: ${{ secrets.TMATE_SERVER_ED25519_FINGERPRINT }}
```

Comment:
this pattern is not very safe, as it exposed the secrets to _all_ steps in the workflow  : to avoid this, best to inject the secrets only in the `debug-wait` step... which means basically inlining the separate action, which does not add any value...

```suggestion
```